### PR TITLE
Investigate Query Selectors

### PR DESCRIPTION
**Pull request by NicoleLatifi**
### Type of change made:
- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Styling- no new features
- [x] Research

### Detailed Description / What does this PR do?

We began commenting on query selectors in regards to scoping and refactoring opportunities. This task needs to be completed.

### Files modified:

src/scripts.js

### Has This Been Tested?
- [x] No
- [ ] Yes

### Are there specific methods for testing?
N/A

### Screenshots (if appropriate):
N/A

### What are the relevant tickets?

#29 #23 

### Message/Questions for reviewer:

Let's keep looking through and commenting on scripts.js!

### Checklist:
- [ ] My code has no unused/commented out code
- [x] My code has no console logs
- [ ] I have performed a self-review of my own code
- [x] I have commented on my code, particularly in hard-to-understand areas

### CODE REVIEW

**Comment by nware1066** [comment]
Great start on this..

File: fitLit_dataModel.js [hunk 1]
```diff
@@ -0,0 +1,123 @@
...
+  sleep() {
+
+  }
+  calculateSleepQualityAverage() {
+    //unshift to put hoursSleptToday at beginning
+  }
+  calculateHoursSleptAverage() {
+    //unshift to put sleepQualityToday at beginning
+  }
+}
```
Comment (Abdeboskey) [inline fitLit_dataModel.js:123]: I'm confused about where this file came from, and why it's here?

File: package-lock.json [hunk 2]
```diff
@@ -24,6 +24,28 @@
         "js-tokens": "^4.0.0"
```
Comment (Abdeboskey) [inline package-lock.json:24]: There are actually a lot of conflicts here, I'm not sure about the best way to resolve this. Let's check in about it this evening!

File: src/scripts.js [hunk 3]
```diff
@@ -286,8 +287,10 @@ user.friendsActivityRecords.forEach(friend => {
   <p class='dropdown-p friends-steps'>${friend.firstName} |  ${friend.totalWeeklySteps}</p>
   `;
 });
+//Put this in a function, locally scope query selector
 
 let friendsStepsParagraphs = document.querySelectorAll('.friends-steps');
+//Should this be with globally scoped query selectors?
 
 friendsStepsParagraphs.forEach(paragraph => {
   if (friendsStepsParagraphs[0] === paragraph) {
```
Comment (Abdeboskey) [inline src/scripts.js:296]: Awesome, it's great to get a better idea of what's happening here.

**Pull request closed without merging**